include/swift/SILOptimizer/Utils/CanonicalOSSALifetime.h

```diff
@@ -269,34 +361,9 @@ class CanonicalizeOSSALifetime {
 
   InstructionDeleter &deleter;
 
-  /// Current copied def for which this state describes the liveness.
-  SILValue currentDef;
-
-  /// Original points in the CFG where the current value's lifetime is consumed
-  /// or destroyed. For guaranteed values it remains empty. A backward walk from
-  /// these blocks must discover all uses on paths that lead to a return or
-  /// throw.
-  ///
-  /// These blocks are not necessarily in the pruned live blocks since
-  /// pruned liveness does not consider destroy_values.
-  SmallSetVector<SILBasicBlock *, 8> consumingBlocks;
-
-  /// Record all interesting debug_value instructions here rather then treating
-  /// them like a normal use. An interesting debug_value is one that may lie
-  /// outisde the pruned liveness at the time it is discovered.
-  llvm::SmallPtrSet<DebugValueInst *, 8> debugValues;
-
   /// Visited set for general def-use traversal that prevents revisiting values.
   GraphNodeWorklist<SILValue, 8> defUseWorklist;
 
-  /// Visited set general CFG traversal that prevents revisiting blocks.
-  GraphNodeWorklist<SILBasicBlock *, 8> blockWorklist;
```

Comment:
@atrick, @eeckstein I tried to not extract out this blockWorklist into CanonicalOSSALifetimeInfo like I did in this patch and instead use a BasicBlockWorklist for the relevant computations. I kept on running into weird behavior and I couldn't figure out what it was. So by moving this around a little bit, I was able to just keep the code as it was. I will mark the function below where I wanted to use BlockWorklist. I didn't have time to follow up with what is up with it.